server/schedulers/utils.go

```diff
@@ -53,14 +53,17 @@ type balancePlan struct {
 
 	sourceScore float64
 	targetScore float64
+
+	excludeTarget []*core.StoreInfo
```

Comment:
The generation and use of `excludeTarget` has nothing to do with `balancePlan`, I think it should be in `balance_region.go`.